tests/test_relperm.py

```diff
@@ -0,0 +1,136 @@
+# test module for the derived RelPerm class located in resqpy.olio.dataframe.py
+
+import os
+import shutil
+import tempfile
+import unittest
```

Comment:
Resqpy currently uses the high-level `pytest` testing framework rather than `unittest`, so for consistency I would suggest switching to pytest.

The syntax is similar:

- The individual unit tests would simply be functions rather than methods within a class.
- Fixtures would replace the "setUp" and "tearDown" methods. You could use the [tmp_path](https://docs.pytest.org/en/6.2.x/tmpdir.html) fixture to get a temporary directory, or the `tmp_model` fixture (which defined in `resqpy/tests/conftest.py`) to get a fresh new model.
- The regular `assert` replaces `self.assertRaises`
- `pytest.raises` replaces `self.assertRaises`
- One does not need the final two lines (unittest.main())

Reply:
I refactored the unit tests to use the pytest framework.